src/test/java/Homework17.java

```diff
@@ -0,0 +1,68 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.awt.*;
+import java.lang.ref.WeakReference;
+import java.util.Collection;
+
+public class Homework17 extends BaseTest {
+
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+
+        String newSongAddedNotificationText = "Added 1 song into \"A51 playlist.\"";
+
+        navigateToPage();
+        provideEmail("jguy1@stny.rr.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        Thread.sleep(2000);
+        searchSong("Reactor");
+        clickViewAllButton();
+        selectFirstSongResult();
+        clickAddSongButton();
+        selectPlaylist();
+
+        Assert.assertEquals(notificationText(), newSongAddedNotificationText);
+
+    }
+
+    public void searchSong(String songTitleKeyword) throws InterruptedException {
+        WebElement searchField = driver.findElement(By.cssSelector("div#searchForm > input[type='search']"));
+        searchField.sendKeys(songTitleKeyword);
+        Thread.sleep(2000);
+    }
+
+    public void clickViewAllButton() throws InterruptedException {
+        WebElement seeAllSearchResults = driver.findElement(By.cssSelector("#searchExcerptsWrapper > div > div > section > h1 > button"));
```

Comment:
Hello,
I have checked your homework 17, you're great. Keep up the good work!

By the way, I sugest you should define selector for element using attribute and use relative path for selector. Avoid making the define selector too long.
//Example: with seeAllSearchResults
`WebElement seeAllSearchResults = driver.findElement(By.cssSelector("button[data-test='view-all-songs-btn']"));`